Update code to use latest renderer code from Shiny v0.7.0

schloerke opened the pull request.

schloerke committed:
Update app to use latest renderer code from Shiny v0.7.0

cpsievert commented on README.md:
```diff
@@ -177,63 +177,59 @@ Now, the `output_tabulator()` function can be called anywhere we want to render
-The `output_transformer` decorator is a decorator factory that takes a function that returns a dictionary of data to be passed to the client side. The function that it decorates is passed two arguments: `_meta` and `_fn`.
+The `Renderer[IT]` class is a decorator that renders output value functions into Jsonifiable objects. A renderer subclass is composed of three main parts: `IT`, `auto_output_ui` and either a `transform` or `render` function.
 
-`_meta` is a dictionary of metadata about the function that is being decorated. We don't use it in our example.
+`IT` represents the type of the value returned by an output value function. It helps users with IDE support know what kind of objects may be returned by their output value function.
 
-`_fn` is the function that is being decorated. Aka the function that goes below the `@render_tabulator()` in your app's server code. In this case we are expecting that that function returns either a pandas dataframe or `None`.
+The `auto_output_ui()` method is used to generate the UI for the output if the renderer were to be used in Express mode. In this case we just use the `output_tabulator()` function we wrote earlier.
```
Update this to reflect any changes made to https://github.com/posit-dev/py-shiny-site/pull/68

schloerke committed:
Updates from blog post

https://github.com/posit-dev/py-shiny-site/pull/75/commits/24dc1514da0e33aa62de536d578651d51b5d73ee and https://github.com/posit-dev/py-shiny-site/pull/75/commits/42fca034b77c4083d44127b482d86ecdfbd4a6e6
